src/menu/views/history_favorites.c

```
@@ -0,0 +1,217 @@
+            VISIBLE_AREA_X0 + FILE_LIST_HIGHLIGHT_WIDTH + LIST_SCROLLBAR_WIDTH,
+            highlight_y + 40,
+            FILE_LIST_HIGHLIGHT_COLOR
+        );
+    }
+
+    char buffer[1024];
+    buffer[0] = 0;
+
+    for(unsigned int i=0; i < item_max; i++) {   
+        if(path_has_value(item_list[i].primary_path)) {
+            sprintf(buffer, "%s%d  : %s\n",buffer ,(i+1), path_last_get(item_list[i].primary_path));
```

Cppcheck (2.10-2)

[error] Undefined behavior

(sprintfOverlappingData)